src/core/sys/darwin/ifaddrs.d

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+
+    D binding for the interface addresses querying
+
+    Defines functions getifaddrs/freeifaddrs and the structure
+    they operate on.
+
+    getifaddrs(3)   get interface addresses
+    freeifaddrs(3)  deallocates the structure returned from getifaddrs
+
+    Copyright:  Copyright (c) 2016 Sociomantic Labs. All rights reserved.
+    License:    $(HTTP www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+    Authors:    Daniel Graczer
+
+*******************************************************************************/
+
+module core.sys.darwin.ifaddrs;
+
+import core.sys.posix.sys.socket;
+
+version (OSX):
```

Comment:
The package name contains `darwin` but the version statement says `OSX`. Please add version statements for other Darwin platforms as well, like this: https://github.com/dlang/druntime/blob/17fb8437d4444e212a1a7c79792052abfcda007a/src/core/sys/darwin/dlfcn.d#L12-L21